docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
```

Comment:
Please be consistent with other pages when putting spaces after section titles (don't know what they do, but this doc is internally inconsistent).

Reply:
Fixed.